ui/docs/how_to_use_the_ui.md

```diff
@@ -0,0 +1,150 @@
+
+---
+
+## Overview of the ARISE Prediction UI
+
+The ARISE Prediction landing page is designed to help users predict resource usage and execution time for AI workloads based on historical data or performance benchmarks. The UI includes:
+
+- Input configuration options.
+- Customizable output parameters.
+- Filtering tools for better result analysis.
+
+---
+
+## Key Sections
+
+- **Prediction Configuration Panel:**  
+  Located in the left sidebar, this panel allows users to define input parameters, output fields, and other configurations required for the prediction process. It also includes the **Predict** button.
+
+- **Input Configuration Panel:**  
+  Enables the selection of input fields. Users can define numeric or categorical fields and set ranges or values to specify the prediction space.
+
+- **Output Configuration Panel:**  
+  Allows the selection of output fields to determine the desired prediction metrics.
+
+- **Main Display Area:**  
+  Displays prediction results and supports additional interactions.
+
+- **Interactive Query Panel:**  
+  Found at the bottom right, this AI Bot lets users ask questions or explore additional features.
+
+---
+
+## Configuring Input Parameters
+
+Input parameters are set in the **Prediction Configuration Panel** and are essential for accurate results. The available fields depend on the dataset and model used. Below are the configurable input parameters for the `MLCommons` dataset:
+
+1. **Number of Accelerators:**  
+   - **Purpose:** Define the number and type of GPUs or accelerators to use.  
+   - **Configuration:** Use the slider to select the number of accelerators (range: 08).
+
+2. **Number of Nodes:**  
+   - **Purpose:** Specify the number of compute nodes.  
+   - **Configuration:** Adjust the slider (range: 14).
+
+3. **Accelerator Type:**  
+   - **Purpose:** Select the type of accelerator.  
+   - **Configuration:** Use the multi-select dropdown for supported models, such as NVIDIA H100.
+
+4. **Host Processor Core Count:**  
+   - **Purpose:** Specify the number of CPU cores available on the host system.  
+   - **Configuration:** Adjust the slider (range: 0128).
+
+5. **Model MLC:**  
+   - **Purpose:** Select the AI model being evaluated.  
+   - **Configuration:** Use the dropdown to choose a model, e.g., `llama2-70b`.
+
+6. **Processor Type:**  
+   - **Purpose:** Define the processor type for computations.  
+   - **Configuration:** Choose from available options, such as Intel Xeon or AMD EPYC.
+
+7. **Scenario:**  
+   - **Purpose:** Define the workload scenario.  
+   - **Configuration:** Select options like *Offline* or *Server*.
+
+---
+
+## Configuring Output Parameters
+
+The **Output Configuration Panel** allows users to select prediction metrics. For the `MLCommons` dataset, available outputs include:
```

Comment:
@eranra link to the mlcommons example folder in the repo

Reply:
fixed